packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -102,25 +103,27 @@ class SmoothProductCardFound extends StatelessWidget {
                         product.productName ??
                             appLocalizations.unknownProductName,
                         overflow: TextOverflow.ellipsis,
-                        style: Theme.of(context).textTheme.headline4,
+                        style: themeData.textTheme.headline4,
                       ),
                       Text(
                         product.brands ?? appLocalizations.unknownBrand,
                         overflow: TextOverflow.ellipsis,
-                        style: Theme.of(context).textTheme.subtitle1,
+                        style: themeData.textTheme.subtitle1,
                       ),
                       Row(
                         children: <Widget>[
                           Icon(
                             Icons.circle,
                             size: 15,
-                            color: helper.getBackgroundColor(),
+                            color: helper.getButtonColor(isDarkMode),
                           ),
                           const Padding(
                               padding: EdgeInsets.only(left: VERY_SMALL_SPACE)),
                           Text(
                             helper.getSubtitle(appLocalizations),
-                            style: Theme.of(context).textTheme.bodyText2,
+                            style: themeData.textTheme.bodyText2!.apply(
+                                color: helper
+                                    .getHeaderForegroundColor(isDarkMode)),
```

Comment:
I guess we need a `getButtonForegroundColor(bool isDarkMode)` then.

Reply:
Changes done.